components/raftstore/src/store/fsm/peer.rs

```diff
@@ -608,10 +609,17 @@ where
                     }
                 }
                 PeerMsg::RaftCommand(cmd) => {
+                    let propose_time = cmd.send_time.saturating_elapsed();
                     self.ctx
                         .raft_metrics
                         .propose_wait_time
-                        .observe(duration_to_sec(cmd.send_time.saturating_elapsed()) as f64);
+                        .observe(duration_to_sec(propose_time) as f64);
```

Comment:
We can just use `propose_time.as_secs_f64()` provided by std.

Reply:
Fixed.